api/api/alogging.py

```diff
@@ -46,15 +49,17 @@ def custom_logging(self, user, remote, method, path, query, body, time, status):
             Required time to compute the request.
         status : int
             Status code of the request.
+        hash_auth_context : str, optional
+            Hash representing the authorization context. Default: ''
         """
 
         self.logger.info(f'{user} '
+                         f'{f"({hash_auth_context}) " if hash_auth_context else ""}'

```

Comment:
Remove redundant f-strings:

```python
>>> a = False
>>> 
>>> f"({a})" if a else ""
>>> ''

>>> a = "hash"
>>> 
>>> f"({a})" if a else ""
'(hash)'
```

Reply:
The f-strings are not redundant but the truth is that the readability could be improved. I have included some changes to do it.